docs/components/base/sensor-controlled.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Configure a Sensor-Controlled Base"
+linkTitle: "sensor-controlled"
+weight: 40
+type: "docs"
+description: "Configure a sensor-controlled base, a robotic base with feedback control from a movement sensor."
+images: ["/icons/components/base.svg"]
+tags: ["base", "components"]
+aliases:
+  - "/components/base/sensor-controlled/"
+# SMEs: Rand H., Martha J.
+---
+
+A `sensor-controlled` base supports a robotic base with feedback control from a movement sensor.
+
+{{% alert title="Requirements" color="note" %}}
+1 or more [movement sensors](/components/movement-sensor/) providing:
+
+- Linear and angular velocity feedback, used by the base's [SetVelocity()](/components/base/#setvelocity) endpoint
+- Orientation feedback, used by the base's [Spin()](/components/base/#spin) endpoint
+  {{% /alert %}}
+
+To configure a `sensor-controlled` base as a component of your machine, first configure the [model of base](/components/base/) you want to wrap with feedback control and each required [movement sensor](/components/movement-sensor/).
+To see what models of movement sensor report which feedback, reference the appropriate column in [Movement Sensor API](/components/movement-sensor/#api).
+
+Configure a `sensor-controlled` base as follows:
+
+{{< tabs name="Configure a Sensor-Controlled Base" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `base` type, then select the `sensor-controlled` model.
+Enter a name for your base and click **Create**.
+
+{{< imgproc src="/components/base/sensor-controlled-base-ui-config.png" alt="An example configuration for a sensor-controlled base in the Viam app config builder" resize="600x" >}}
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    { ... INSERT MOVEMENT SENSOR CONFIGURATION },
+    { ... INSERT BASE CONFIGURATION },
+    {
+      "name": "my-sensor-controlled-base",
+      "model": "sensor-controlled",
+      "type": "base",
+      "namespace": "rdk",
+      "attributes": {
+        "movement_sensor": [
+          "<your-orientation-or-velocity-movement-sensor-1>",
+          "<your-orientation-or-velocity-movement-sensor-2>"
+        ],
+        "base": "<your-base>"
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `sensor-controlled` bases:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `movement-sensor` | array | **Required** | Array with the `name` of any movement sensors on your base you want to gather feedback from. |
```

Comment:
```suggestion
| `movement-sensor` | array | **Required** | Array with the `name` of any movement sensors on your base you want to gather feedback from. The driver will select the first movement sensor with the properties that will allow it to get feedback for either of the SetVelocity or Spin APIS|
```

Not sure if this is the right place for it, same as `merged` movement sensor, we select the first movement sensor in the array with the appropriate properties for the SetVelocity and Spin calls. 

So you can have two movement sensors in the array, 
`movement_sensor: [velly spinny]
the first - let's call it `velly` that has `true` returned for `LinearVelocitySupported` and `AngularVelocitySupported` in its properties call but `false` for its `OrientationSupported` property; the second - let's call it `spinny` returns  `true` only for it's `OrientationSupported` property. 

`sensor-controlled` base will use `velly` for feedback in its `SetVelocity` call and `spinny` for feedback in it's spin call.

Reply:
That makes sense! Rephrased a bit because they probably won't have context for properties, that ok or do you want original phrasing?